back-end/aws-cloud9-api/src/main/java/com/serverless/dal/ClosedQuestion.java

```diff
@@ -0,0 +1,94 @@
+package com.serverless.dal;
+
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBAttribute;
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBAutoGeneratedKey;
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBHashKey;
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBRangeKey;
+
+public class ClosedQuestion extends Question {
+	
+		private String id;
+		private String name;
+		private String question;
+		private String form_membership;
+	    private String correct_answer;
+	    private String wrong_answer1;
+	    private String wrong_answer2;
+	    private String wrong_answer3;
```

Comment:
Tak tego na pewno nie robimy. Lista możliwych odpowiedzi powinna być elastyczna, nie zawsze to muszą być 4 możliwe odpowiedzi. Powinna tu być mapa map<String, String>, a zmienna z poprawną powinna przechowywać klucz tej mapy do poprawnej odpowiedzi.